calc/src/App.js

```diff
@@ -0,0 +1,148 @@
+import React, { Component } from "react";
+import logo from "./logo.svg";
+// import "./App.css";
+import "./style.scss";
+class App extends Component {
+  state = {
+    calorie: 0,
+    done: false
+  };
+
+  calculate = e => {
+    e.preventDefault();
+    let calorieCalc;
+    let finalCalc = 0;
+    const height =
+      Number(e.target.inches.value) + Number(e.target.feet.value) * 12;
```

Comment:
`Number()` is discouraged since it's actually not the same as primitive numbers. Do `+e.target.inches.value` instead.